src/Surfnet/ServiceProviderDashboard/Application/Service/AttributeService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Service;
+
+use Surfnet\ServiceProviderDashboard\Domain\ValueObject\Attribute;
+use Surfnet\ServiceProviderDashboard\Legacy\Repository\AttributesMetadataRepository;
+
+class AttributeService implements AttributeServiceInterface
+{
+    /**
+     * @var AttributesMetadataRepository
+     */
```

Comment:
We need to decouple from this legacy Attributes Metadata Repo. Feel free to create a new one and leave the present one where it is for now.